pilot/pkg/networking/core/cluster_builder_test.go

```diff
@@ -1410,8 +1410,8 @@ func TestClusterDnsLookupFamily(t *testing.T) {
 			if c.DnsLookupFamily != tt.expectedFamily {
 				t.Errorf("Unexpected DnsLookupFamily, got: %v, want: %v", c.DnsLookupFamily, tt.expectedFamily)
 			}
-			if c.DnsJitter.AsDuration() <= 0 {
-				t.Errorf("Unexpected DnsJitter, got: %v, want: positive", c.DnsJitter.AsDuration())
+			if c.DnsJitter.AsDuration() < 0 {
+				t.Errorf("Unexpected DnsJitter, got: %v, want: >= 0", c.DnsJitter.AsDuration())
```

Comment:
Doesn't 0 disable jitter? I'm confused by what we are testing here with this check and the expectation of having `>= 0`. Can you clarify what's exactly being tested here?

Reply:
It checks if it was set to a valid value. Its a pretty basic test.

Reply:
Yeah makes sense now that 0 is a valid option. This test was just updated to reflect that.